addons/l10n_ae/data/fiscal_templates_data.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="account_fiscal_position_uae_countries" model="account.fiscal.position.template">
+        <field name="name">Fiscal Position for UAE Countries</field>
+        <field name="auto_apply" eval="True"/>
+        <field name="sequence" eval="1"/>
+        <field name="country_id" ref="base.ae"/>
+        <field name="chart_template_id" ref="uae_chart_template_standard"/>
+    </record>
+
+    <record id="account_fiscal_position_non_uae_countries" model="account.fiscal.position.template">
+        <field name="name">Fiscal Position for Non-UAE Countries</field>
```

Comment:
It is a bit weird to use Fiscal Position for... in the name.  We could say UAE vs Non-UAE.